5/README.md

```diff
@@ -158,9 +158,9 @@ Specifies a menu, menu data structure, contents, products and so on
   | key:string
   | value:opencannabis.products.Plant]
```

Comment:
well, the changes performed now make this wrong. **.products.Plant** should be **.products.Preroll**

Reply:
DANG

Reply:
Good catch